flags/application.json

```diff
@@ -14,6 +14,26 @@
     "shift": 4,
     "undocumented": true
   },
+  "RPC_PRIVATE_BETA": {
+    "description": "Application has access to the RPC HTTP server",
```

Comment:
Possibly replace `RPC HTTP server` with `RPC API` ?

Reply:
Apps can still interact with the RPC API using the local IPC server, the HTTP server hosted by the desktop client is locked to the beta though

Reply:
RPC API has more functionality than IPC

Reply:
RPC != IPC